meilisearch-release-changelog/v1.13.0.md

```diff
@@ -0,0 +1,76 @@
+# v1.13.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+> [!WARNING]
+> Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.13 introduces several significant improvements: our Vector Store feature is now fully stabilized for seamless hybrid search integration, dumpless version upgrades eliminate the need for reindexing during updates, granular filterable attribute settings allow for optimized indexing performance, and a new experimental feature enables distributed search queries across multiple Meilisearch instances to better balance network loads.
```

Comment:
```suggestion
Meilisearch v1.13 introduces several significant improvements including the stabilization of the Vector Store feature for seamless hybrid search integration! 🎉 This version also ease the Meilisearch upgrade by simplifying the process: you don't need dump for migrating anymore! Finally, we introduced more granular filterable attribute settings to optimize your indexing performance.
```

Reply:
I suggest something in multiple sentences, easier to read, and more focus on the value for the users than just listing the features 😊

Reply:
Proxy search request are important but not a important feature enough to be put here. It would just make the things heavier